src/utils/filters/resource-filter-builder/resourceFilterBuilder.spec.js

```diff
@@ -0,0 +1,31 @@
+import dayjs from 'dayjs';
+import { formatDateTimeISO } from '../../time-entry-utils/timeEntryUtils';
+import { filterOnOrAfterDate, filterBeforeDate } from './resourceFilterBuilder';
+
+const date = '2022-10-29';
+const dateTime = date + ' 19:18:09';
+const resourceDateProperty = 'startTime';
+
+describe('resourceFilterBuilder', () => {
+  describe('filterOnOrAfterDate', () => {
+    it('should create a filter specifying that the resource property is greater or equal to midnight of the date', () => {
+      expect(filterOnOrAfterDate(resourceDateProperty, dateTime)).toEqual(
+        resourceDateProperty +
+          ">='" +
+          formatDateTimeISO(dayjs(date + ' 00:00:00')) +
+          "'"
+      );
```

Comment:
Same as before in the tests in this file with explicitly checking the string instead of calling `formatDateTimeISO`

Reply:
Will update according to discussion in the first/main comment.

Reply:
updated